.github/workflows/conda.yml

```diff
@@ -1,32 +1,182 @@
+          - {os: windows-2022, os-family: windows}
+          - {os: macos-11, os-family: macos}
+          - {os: macos-12, os-family: macos}
+          - {os: macos-13, os-family: macos}
+    runs-on: ${{ matrix.env.os }}
+    steps:
+      - name: Install Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          miniconda-version: latest  # needed for macOS 13
+          python-version: ${{ matrix.python-version }}
+      - name: Download Conda Package Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: khiops-conda-${{ matrix.env.os-family }}
+          path: khiops-conda
+      - name: Install the Conda package (Windows)
+        if: runner.os == 'Windows'
+        run: conda install -c ./khiops-conda/ khiops-core
+      # In Linux/macOS we need the conda-forge channel to install their pinned versions
+      - name: Install the Conda package (Linux/macOS)
+        if: runner.os != 'Windows'
+        run: conda install -c conda-forge -c ./khiops-conda/ khiops-core
+      - name: Test that the executables are installed
+        run: |
+          MODL -v
+          MODL_Coclustering -v
+  # Release is only executed on tags
+  # Note: For this job to work the secrets variables KHIOPS_ANACONDA_CHANNEL_TOKEN and
+  #       KHIOPS_DEV_ANACONDA_CHANNEL_TOKEN must be set with valid anaconda.org access tokens
+  release:
+    if: github.ref_type == 'tag'
+    needs: test
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+    steps:
+      - name: Download package artifacts
+        uses: actions/download-artifact@v4
+        with:
+          # See the upload-artifact step in the build job for the explanation of this pattern
+          path: ./khiops-conda
+          pattern: khiops-conda-*
+          merge-multiple: true
+      - name: Install Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          miniconda-version: latest
+          python-version: '3.12'
+      - name: Install requirement packages
+        run: conda install -y anaconda-client conda-index
+      - name: Reindex the package directory
+        run: python -m conda_index ./khiops-conda
+      - name: Upload the packages to anaconda.org
+        run: |
+          # Set the anaconda.org channel
+          ANACONDA_CHANNEL="${{ inputs.release-channel || 'khiops-dev' }}"
+
+          # For the release channel: upload without forcing
+          if [[ "$ANACONDA_CHANNEL" == "khiops" ]]
+          then
+            anaconda --token "${{ secrets.KHIOPS_ANACONDA_CHANNEL_TOKEN }}" upload \
+              --user "$ANACONDA_CHANNEL" ./khiops-conda/*/*.tar.bz2
+          # For the dev channel: upload with forcing
+          else
+            anaconda --token "${{ secrets.KHIOPS_DEV_ANACONDA_CHANNEL_TOKEN }}" upload \
+              --user "$ANACONDA_CHANNEL" --force ./khiops-conda/*/*.tar.bz2
+          fi
+      - name: Extract package version
```

Comment:
I don't understand the last part, why do we have the zip release and upload. Perhaps a short call to clarify it for me.

Reply:
Yeah, it is a bit redundant. At some point we used the zip for the tests. I'll eliminate it, the artifacts for each platform are available anyway at each run of the workflow.

Reply:
In second thought i'll just eliminate the upload to the release. The zip maybe useful because it contains the packages for all platforms in a single (local) conda channel. That maybe useful for tests.